### PR TITLE
Remove explicit require json/pure from cache_instance_list.rb

### PR DESCRIPTION
**Pull request by somic**
Newer versions of fog pull in fog-json which will take care of figuring out the best way how to work with json. This lets it do its job, no need to hardcode json/pure.

### CODE REVIEW

**Comment by somic** [comment]
Some AWS regions need fog newer than is available from rubygem-fog. Puppet-omnibus already gives us that so let's use it.

File: manifests/aws_prune.pp [hunk 1]
```diff
@@ -53,6 +53,7 @@
      tip         => 'talk to kwa',
```
Comment (solarkennedy) [inline manifests/aws_prune.pp]: You might have better ideas on what to say here

**Comment by solarkennedy** [comment]
Ship! Great job!

**Comment by bobtfish** [comment]
:+1:

**Pull request merged**